src/components/AccountDetails.tsx

```diff
@@ -40,8 +41,21 @@ export default function AccountDetail({
   const ethBalanceFormatted = commify(
     formatBNToString(tokenBalances?.ETH || Zero, 18, 6),
   )
+  const [user, setUser] = useState<string>("")
```

Comment:
"user" is pretty vague, can we call this "udUser" or at least add a comment indicating that it pertains to unstoppable domains?